examples/step-74/step-74.cc

```diff
@@ -509,19 +509,21 @@ namespace Step74
           for (unsigned int i = 0; i < n_dofs_face; ++i)
             for (unsigned int j = 0; j < n_dofs_face; ++j)
               copy_data_face.cell_matrix(i, j) +=
-                (-diffusion_coefficient *              // - nu
-                   fe_iv.jump(i, point) *              // [v_h]
-                   (fe_iv.average_gradient(j, point) * // ({grad u_h} .
-                    normals[point])                    //  n)
-
-                 - diffusion_coefficient *               // - nu
-                     (fe_iv.average_gradient(i, point) * // (grad v_h .
-                      normals[point]) *                  //  n)
-                     fe_iv.jump(j, point)                // [u_h]
-
-                 + diffusion_coefficient * penalty * // + nu sigma
-                     fe_iv.jump(i, point) *          // [v_h]
-                     fe_iv.jump(j, point)            // [u_h]
+                (-diffusion_coefficient *                 // - nu
+                   fe_iv.jump_in_shape_values(i, point) * // [v_h]
+                   (fe_iv.average_of_shape_gradients(j,
```

Comment:
This looks quite ugly now. :-( Can you try to improve the layout? At least align the comments, please.
I think you can move the opening ( in the first line on a separate line.

Reply:
I agree. Unfortunately, without a non-lint directive, the indenter is insistent on touching these lines (the comment just goes over the line width limit). Here are some alternatives that I've managed to produce thus far. I'll try to fiddle around more a bit later.

Option 1: (the initial changed proposed in the PR)

Option 2: Aligned comments
```
              copy_data_face.cell_matrix(i, j) +=
                (-diffusion_coefficient *                     // - nu
                   fe_iv.jump_in_shape_values(i, point) *     // [v_h]
                   (fe_iv.average_of_shape_gradients(j,       //
                                                     point) * // ({grad u_h} .
                    normals[point])                           //  n)
```
Option 3: Line break over arguments, and aligned comments
```
              copy_data_face.cell_matrix(i, j) +=
                (

                  -diffusion_coefficient *                 // - nu
                    fe_iv.jump_in_shape_values(i, point) * // [v_h]
                    (fe_iv.average_of_shape_gradients(     //
                       j,                                  //
                       point) *                            // ({grad u_h} .
                     normals[point])                       //  n)
```

Reply:
I think option 2 is fine.

Reply:
Ok, I've gone for option 2 :-)